modules/netIdSystem.js

```diff
@@ -18,22 +23,108 @@ export const netIdSubmodule = {
    * decode the stored id value for passing to bid requests
    * @function decode
    * @param {(Object|string)} value
+   * @param {SubmoduleConfig|undefined} config
    * @returns {(Object|undefined)}
    */
-  decode(value) {
-    return (value && typeof value['netId'] === 'string') ? { 'netId': value['netId'] } : undefined;
+  decode(value, config) {
+    let netId;
+    if (value && typeof value['netId'] === 'string') {
+      netId = { 'netId': value['netId'] };
+    } else {
+      return undefined;
+    }
+    // check for A/B testing configuration and hide ID if in Control Group
+    const abConfig = checkAbTestingConfig(config, netId);
+    if (abConfig.enabled === true && abConfig.controlGroup === true) {
```

Comment:
Can the enabled property be checked to be true in `checkAbTestingConfig` instead of happening twice here?